pseudo_label.py

```diff
@@ -1,21 +1,61 @@
+# Copyright (c) 2021 Ichiro ITS
+#
+# Permission is hereby granted, free of charge, to any person obtaining a copy
+# of this software and associated documentation files (the "Software"), to deal
+# in the Software without restriction, including without limitation the rights
+# to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+# copies of the Software, and to permit persons to whom the Software is
+# furnished to do so, subject to the following conditions:
+#
+# The above copyright notice and this permission notice shall be included in
+# all copies or substantial portions of the Software.
+#
+# THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+# IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+# FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL
+# THE AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+# LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+# OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
+# THE SOFTWARE.
+
+import argparse
+from ctypes.wintypes import BOOL
 import cv2
 import os
 from tqdm import tqdm
 
 from yolo import Yolo
 
 if __name__ == "__main__":
-  # TODO: change to args
-  data_path = "data"
-  weights = os.path.join(data_path, "yolo_weights.weights")
-  config = os.path.join(data_path, "config.cfg")
-  labels = os.path.join(data_path, "obj.names")
-  draw_output = True
+  parser = argparse.ArgumentParser()
+  parser.add_argument('--CONF_THRESH', help='threshold for object confident',
+                      type=float, default=0.4)
+  parser.add_argument('--config', help='path for yolo config',
+                      type=str, default="data/config.cfg")
+  parser.add_argument('--CUDA', help='threshold for Non-maximum Suppression',
+                      type=bool, default=False)
```

Comment:
I don't think you write a correct help here

Reply:
ya I will fix it